src/app/layout/components/header/header.component.html

```diff
@@ -8,15 +8,28 @@ <h1 class="header-logo__text">IT-House</h1>
       <ul class="header-nav">
         <li><a class="header-nav__link" routerLink="/posts">Новости</a></li>
         <li><a class="header-nav__link" routerLink="/events">Мероприятия</a></li>
-        <div class="header-wrapper" *ngIf="isLogged">
-          <li><a class="header-nav__link" routerLink="">Поддержка</a></li>
-          <li><a class="header-nav__link" routerLink="">Обратная связь</a></li>
+        <div class="header-wrapper">
+          <li><a class="header-nav__link" routerLink="/support">Обратная связь</a></li>
           <li><a class="header-nav__link" routerLink="">Профиль</a></li>
         </div>
-        <div class="header-wrapper" *ngIf="!isLogged">
+      </ul>
+    </nav>
+  </div>
+</header>
+
+<header class="header" *ngIf="!isLogged">
```

Comment:
@k1k8r lets substitute `<header class="header" *ngIf="!isLogged">` to `<ng-template #unauthorised>` for example. It allows don't trigger `isLogged` variable twice.

`<header class="header" *ngIf="isLogged; else unauthorised">`